pkg/views/profile/create.go

```diff
@@ -55,6 +56,12 @@ func ProfileCreationView(c *config.Config, profileAddView *ProfileAddView, editi
 					if str == "" {
 						return errors.New("server API URL can not be blank")
 					}
+
+					_, err := url.ParseRequestURI(str)
+					if err != nil {
+						return errors.New("invalid url, must be of http/https/ssh format")
```

Comment:
Remove `ssh` from the string. `ssh` is not an acceptable protocol here.

Reply:
fixed.